src/cogs/fun.py

```
@@ -0,0 +1,47 @@
+import random
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)